docs/README.md

```diff
@@ -72,8 +72,8 @@ This can be accomplished by running the `make dev_install` command as outlined i
 To check the documentation for formatting issues, use the following:
 
 ```bash
-## Lints all content, applies lint autofixes and prettier changes
-yarn lint
+## Format files with `prettier`
+yarn format
 
 ## Lints documentation content using Vale Server
```

Comment:
I realize this isn't part of this PR, but you might as well remove this Vale section, since we don't use it in the build process or really recommend running it locally, since it's so noisy -- `yarn vale /path/to/file` appears to run vale on all docs, not just the individual file.

Reply:
Removed, thanks.